docs/Available-templates-for-dotnet-new.md

```diff
@@ -69,4 +70,4 @@ Here is a list (alpha sorted) of templates which are available for use with `dot
 | Name     | Quick Install |
 |----------|:--------------|
 | [GtkSharp](https://github.com/GtkSharp/GtkSharp) | `dotnet new -i "GtkSharp.Template.VBNet"` |
-| [InteXX Assorted Templates](https://github.com/InteXX/Templates) | `dotnet new -i "Intexx.Templates"` |
\ No newline at end of file
+| [InteXX Assorted Templates](https://github.com/InteXX/Templates) | `dotnet new -i "Intexx.Templates"` |
```

Comment:
This line should not be changed (git shows \ No newline at end of file)

Reply:
Amended. I think it was a result of using the online editor.